src/synth/drumModules/snareDrum.js

```diff
@@ -32,7 +32,7 @@ export default function (audioCtx, destination, time, { level, tone, snappy }) {
   outputVCA.amplitude.value = outputLevel;
 
   // envelopes
-  const noiseEnv = new ADGenerator(LINEAR, 0.1, 75, 0, 0.5);
+  const noiseEnv = new ADGenerator(LINEAR, 0.1, 160, 0, 0.9);
```

Comment:
If you look at the waveforms you can see that the original 808 sample trails off much longer, around  260ms 

![808-sample-snare](https://cloud.githubusercontent.com/assets/6077905/21626441/1562a7ec-d1df-11e6-9ba1-323ec93928e8.jpg)

vs your snare  around 130ms 
![io-808-snare](https://cloud.githubusercontent.com/assets/6077905/21626446/19613c3c-d1df-11e6-909e-6693cc66e389.jpg)